packages/smooth_app/lib/pages/product/edit_image_button.dart

```diff
@@ -9,12 +9,21 @@ class EditImageButton extends StatelessWidget {
     required this.label,
     required this.onPressed,
     this.borderWidth,
-  });
+  }) : _centerContent = false;
+
+  /// Centered version of the button.
+  const EditImageButton.center({
+    required this.iconData,
+    required this.label,
+    required this.onPressed,
+    this.borderWidth,
+  }) : _centerContent = true;
```

Comment:
Wouldn't a unique constructor with `this.centerContent = false` be easier to maintain?